exporters/elasticsearch-exporter/src/test/resources/dmn/decision-table.dmn

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" id="force-users" name="Force Users" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.11.0">
-  <decision id="jedi-or-sith" name="Jedi or Sith">
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" id="force-users" name="Force Users" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.12.0">
+  <decision id="jedi_or_sith" name="Jedi or Sith">
```

Comment:
💭 I was wondering whether there are any other cases of `jedi-or-sith` left: yes there are 3 .dmn files (`decision-table_v2.dmn`, `decision-table.dmn` and `decision-table-with-renamed-drg`) and 1 test (`DeploymentDmnTest`) that still uses it. Doesn't have to be done in this PR, but just wanted to highlight it.